x-pack/plugins/security_solution/public/common/components/landing_page/get_started/hooks/use_user_name.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { useEffect, useState } from 'react';
+import { useKibana } from '../../../../lib/kibana';
+
+export const useUserName = () => {
```

Comment:
does this need to be it's own hook? x-pack/plugins/security_solution/public/common/lib/kibana/hooks.ts has much the same code in useCurrentUser

Reply:
Good catch, I'll use useCurrentUser